src/Microsoft.ML.Transforms/CategoricalCatalog.cs

```diff
@@ -43,13 +43,17 @@ public static class CategoricalCatalog
         /// <param name="catalog">The transform catalog</param>
         /// <param name="inputColumn">The input column</param>
         /// <param name="outputColumn">The output column. If <c>null</c>, <paramref name="inputColumn"/> is used.</param>
+        /// <param name="hashBits">Number of bits to hash into. Must be between 1 and 30, inclusive.</param>
+        /// <param name="invertHash">Limit the number of keys used to generate the slot name to this many. 0 means no invert hashing, -1 means no limit.</param>
         /// <param name="outputKind">The conversion mode.</param>
         /// <returns></returns>
         public static OneHotHashEncodingEstimator OneHotHashEncoding(this TransformsCatalog.CategoricalTransforms catalog,
                 string inputColumn,
                 string outputColumn = null,
+                int hashBits = OneHotHashEncodingEstimator.Defaults.HashBits,
```

Comment:
>hashBits [](start = 20, length = 8)

Could you please add a test to `test\Microsoft.ML.Tests\Transformers\CategoricalTests.cs` for this constructor?